src/js/contentscript.js

```diff
@@ -2,370 +2,121 @@ import 'expose?$!expose?jQuery!jquery/dist/jquery.min.js';
 import 'expose?Vue!vue/dist/vue.min.js'; // Can't just import Vue from 'vue' because of async dependencies
 import VueI18n from 'vue-i18n';
 
-import template from 'html!../templates/panel.html';
 import RR from './rr';
-import RRLocales from './i18n/locales.js';
+import RRlocales from './i18n/locales.js';
```

Comment:
was a typo, I've created the import and wrongly typed the name, but yes it must be `RRLocales`